### PR TITLE
modified notes about excluded crds

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-12297

### CODE REVIEW

File: business_continuity/backup_restore/backup_arch.adoc [hunk 1]
```diff
@@ -22,11 +22,11 @@ View the following ordered list of the cluster backup and restore process for id
...
+- Exclude all resources that are a part of the included API groups, but are either not needed or are being recreated by owner resources that are also backed up: `clustermanagementaddon.addon.open-cluster-management.io` ,`backupschedule.cluster.open-cluster-management.io`, `restore.cluster.open-cluster-management.io`, `clusterclaim.cluster.open-cluster-management.io`, `discoveredcluster.discovery.open-cluster-management.io`.
 
 - Back up secrets and ConfigMaps with one of the following labels: `cluster.open-cluster-management.io/type`, `hive.openshift.io/secret-type`, `cluster.open-cluster-management.io/backup`. 
 
-- Use the `cluster.open-cluster-management.io/backup` label for any other resources that you want to be backed up and are not included in the previously mentioned criteria. See the following example:
+- Use the `cluster.open-cluster-management.io/backup` label for any other resources that you want to be backed up and are not included in the previously mentioned criteria or are not part of the excluded API groups. See the following example:
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_arch.adoc]: update : or are not part of the excluded API groups. See the following example:

with : or are part of the excluded API groups. See the following example:

**Review by birsanv — CHANGES_REQUESTED** [review]

**Review by birsanv — APPROVED** [review]
lgtm

File: business_continuity/backup_restore/backup_arch.adoc [hunk 2]
```diff
@@ -22,11 +22,11 @@ View the following ordered list of the cluster backup and restore process for id
...
 
-- Exclude all resources that are a part of the included API groups, but are either not needed or are being recreated by owner resources, which are also backed up: `clustermanagementaddon`, `observabilityaddon`, `applicationmanager`, `certpolicycontroller`, `iampolicycontroller`, `policycontroller`, `searchcollector`, `workmanager`, `backupschedule`, `restore`, `clusterclaim.cluster.open-cluster-management.io`.
+- Exclude all resources that are a part of the included API groups, but are either not needed or are being recreated by owner resources that are also backed up: `clustermanagementaddon.addon.open-cluster-management.io` ,`backupschedule.cluster.open-cluster-management.io`, `restore.cluster.open-cluster-management.io`, `clusterclaim.cluster.open-cluster-management.io`, `discoveredcluster.discovery.open-cluster-management.io`.
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_arch.adoc]:
```suggestion
- Exclude all of the following resources that are a part of the included API groups, but are either not needed or are being recreated by owner resources that are also backed up: `clustermanagementaddon.addon.open-cluster-management.io` ,`backupschedule.cluster.open-cluster-management.io`, `restore.cluster.open-cluster-management.io`, `clusterclaim.cluster.open-cluster-management.io`, `discoveredcluster.discovery.open-cluster-management.io`.
```

File: business_continuity/backup_restore/backup_arch.adoc [hunk 3]
```diff
@@ -22,11 +22,11 @@ View the following ordered list of the cluster backup and restore process for id
...
 
 - Back up secrets and ConfigMaps with one of the following labels: `cluster.open-cluster-management.io/type`, `hive.openshift.io/secret-type`, `cluster.open-cluster-management.io/backup`. 
 
-- Use the `cluster.open-cluster-management.io/backup` label for any other resources that you want to be backed up and are not included in the previously mentioned criteria. See the following example:
+- Use the `cluster.open-cluster-management.io/backup` label for any other resources that you want to be backed up and are not included in the previously mentioned criteria or are part of the excluded API groups. See the following example:
 +
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_arch.adoc:31]:
```suggestion

+
```

Comment (dockerymick) [reply business_continuity/backup_restore/backup_arch.adoc:31]: @jc-berger will you also add a new line break on line 44?

**Review by dockerymick — CHANGES_REQUESTED** [review]

**Review by dockerymick — APPROVED** [review]
Thanks Jake, will you add a break line on 44: https://github.com/stolostron/rhacm-docs/pull/6573/files#r1657311846

Otherwise, this looks good

**Pull request merged**